Source/Scene/Cesium3DTileset.js

```diff
@@ -344,22 +354,25 @@ define([
         if (!outOfCore) {
             return;
         }
-        if (!hasAvailableRequests(tiles3D)) {
+        if (!hasAvailableRequests(tile)) {
             return;
         }
 
-        var stats = tiles3D._statistics;
-        ++stats.numberOfPendingRequests;
-        addLoadProgressEvent(tiles3D);
-
         tile.requestContent();
-        var removeFunction = removeFromProcessingQueue(tiles3D, tile);
-        when(tile.processingPromise).then(addToProcessingQueue(tiles3D, tile)).otherwise(removeFunction);
-        when(tile.readyPromise).then(removeFunction).otherwise(removeFunction);
+
+        if (!tile.isContentUnloaded()) {
+            var stats = tiles3D._statistics;
+            ++stats.numberOfPendingRequests;
+            addLoadProgressEvent(tiles3D);
```

Comment:
This call keeps track of the number of requests so it is possible that it could increase the number of pending requests here even though the scheduler doesn't make the request.

At one point we talked about adding an event (or promise or whatever) that fires when the request is actually made (or not).  Perhaps we should use that to call `addLoadProgressEvent`.

Note that this should all happen before events are raised at the end of the frame with `frameState.afterRender`.